test/junit/scala/tools/nsc/settings/TargetTest.scala

```diff
@@ -50,8 +57,10 @@ class TargetTest {
     check("-target:jvm-12", "12")
     check("-target:12", "12")
 
-    checkFail("-target:jvm-13") // not yet...
-    checkFail("-target:msil") // really?
+    checkFail("-target:jvm-6")    // no longer
+    checkFail("-target:jvm-13")   // not yet...
```

Comment:
Uncomfortably soon!

Reply:
Yeah I decided not to tweak it yet. Seth needs stuff to do later, probably as part of the community build.